### PR TITLE
Fix flaky yaml spec tests

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

**Comment by eddyxu** [comment]
Re-triggered the tests 3 times, all passed.

File: python/rikai/spark/sql/codegen/pytorch.py [hunk 1]
```diff
@@ -24,6 +25,9 @@
 from rikai.io import open_uri
 from rikai.torch.pandas import PandasDataset
 
+DEFAULT_NUM_WORKERS = 8
```
Comment (changhiskhan) [inline python/rikai/spark/sql/codegen/pytorch.py:28]: nit: maybe add this to rikai.options?

**Review by changhiskhan — APPROVED** [review]
good stuff

File: python/rikai/spark/sql/codegen/pytorch.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/spark/sql/codegen/pytorch.py:28]: will do later

**Pull request merged**